{
  "id": "uno-ooo-parser",
  "version": "0.1.28",
  "libre_office_ver": "7.2",
  "name": "AuthenticationFallbackRequest",
  "type": "exception",
  "namespace": "com.sun.star.ucb",
  "parser_args": {
    "sort": false,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "allow_db": true,
    "from_imports": [
      [
        "..task.classified_interaction_request",
        "ClassifiedInteractionRequest",
        "ClassifiedInteractionRequest_9f72121b"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.task.ClassifiedInteractionRequest": "ClassifiedInteractionRequest_9f72121b"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "full_imports": {
      "general": [
        "com.sun.star.task.ClassifiedInteractionRequest"
      ],
      "typing": []
    },
    "name": "AuthenticationFallbackRequest",
    "imports": [],
    "namespace": "com.sun.star.ucb",
    "extends": [
      "com.sun.star.task.ClassifiedInteractionRequest"
    ],
    "desc": [
      "An interaction continuation handing back some authentication data.",
      "",
      "**since**",
      "",
      "    LibreOffice 4.4"
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1ucb_1_1AuthenticationFallbackRequest.html",
    "items": {
      "properties": [
        {
          "name": "instructions",
          "returns": "str",
          "origtype": null,
          "origin": "string",
          "desc": [
            "Instructions to be followed by the user."
          ],
          "raises_get": "",
          "raises_set": ""
        },
        {
          "name": "url",
          "returns": "str",
          "origtype": null,
          "origin": "string",
          "desc": [
            "url to be opened in browser"
          ],
          "raises_get": "",
          "raises_set": ""
        }
      ]
    }
  }
}